atrain_match/statistics/orrb_CPH_stat.py

```diff
@@ -37,6 +37,8 @@ def do_stats(self):
         n_water_water_cal = self.ac_data["n_water_water_cal"]
         Num = self.ac_data["Num"]
 
+        bias_cal = np.divide(1. * (n_ice_water_cal - n_water_ice_cal), Num)
```

Comment:
I actually copied it blind from `orrb_CFC_stat.py` without thinking. However, thinking about it now a plus might be correct. I mean that ... I would interpret the bias for a binary classification as the "Misclassification rate" or "Error rate" which can be calculated as `(false_positives+false_negatives) / N` so the plus could be right. For this case both `orrb_CFC_stat.py` and `orrb_CPH_stat.py` must be corrected. Would you aggree ?

Reply:
Had a look at this issue recently when I had some time left. I think the minus sign is right. Because:

`Bias = 1/N * sum(x_ref - x) =    1/N * sum(x_ref)  -  1/N * sum(x)`

The first term is just the mean of the reference, the second of the observation. 

Now assuming a contigency table for cloud phase e.g.:

```
                  Reference
...............| ICE |  LIQ  |
Obs        ------------------
           ICE |  A   |  B   |
           ------------------
           LIQ |  C   |  D   |
           ---------------------
```

Mean of reference (Liquid cloud fraction): `(B+D) / N`
Mean of obs (Liquid cloud fraction): `(C+D)/N`

Then the bias is:
`(B+D)/N - (C+D)/N = 1/N * [(B+D) - (C+D)] = 1/N * (B+D-C-D) = (B-C)/N`

That is what I also coded in the scores.py file.